client/src/legacy/TinyMCE_sslink-anchor.js

```diff
@@ -44,7 +44,7 @@ const plugin = {
 };
 
 const modalId = 'insert-link__dialog-wrapper--anchor';
-const sectionConfigKey = 'SilverStripe\\CMS\\Controllers\\CMSPageEditController';
+const sectionConfigKey = 'SilverStripe\\Admin\\LeftAndMain';
```

Comment:
`LeftAndMain` now dynamically lists all link forms registered with `ModalController.link_modal_form_factories`

These do get inherited, so `CMSPageEditController` would still work, but it seems sensible to use the more generic controller here (especially since `CMSPageEditController` is not long for this world)